app/uploaders/banner_image_uploader.rb

```diff
@@ -0,0 +1,8 @@
+class BannerImageUploader < CarrierWave::Uploader::Base
+  include CarrierWave::Compatibility::Paperclip
+
+  # Define valid extensions for site banner image
+  def extension_white_list
+    %w(jpg jpeg png gif tif tiff)
```

Comment:
@mjgiarlo RE: dimensions... That's an area of question for me.  The current default (in Sufia) is an image that's 5000x1000 ([this one](https://cloud.githubusercontent.com/assets/92044/18370978/88ecac20-75f6-11e6-8399-6536640ef695.jpg)).  But, the [Lerna sample site](https://ggeisler.github.io/lerna-dashboard/sdr-home) has an image that is smaller, 2206x446 ([this stanford one](https://ggeisler.github.io/lerna-dashboard/sdr/stanford-oval-blurred.png)).  Either size seems to be "valid".

Currently, in my edit UI, I'm stating ["2000x500 (or larger)"](https://github.com/projecthydra-labs/lerna/pull/402/files#diff-6726cc707ff0afef2e3c00a03afeaa89R28) but have no code validation in place.  So, I guess the question in my mind is.. do we need to auto-resize, or should we just check if dimensions are *at least* a particular size?